File: calc/app/src/main/java/com/example/calc/MainActivity.java
package com.example.calc;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    EditText edtvalu1,edtvalu2;
    TextView txtResult;
    Button btn1,btn2,btn3,btn4;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

       intview();
    }

    private void intview()
    {
        edtvalu1=findViewById(R.id.edtvalu1);
        edtvalu2=findViewById(R.id.edtvalue2);
        txtResult=findViewById(R.id.txtresult);
        btn1=findViewById(R.id.btn1);
        btn2=findViewById(R.id.btn2);
        btn3=findViewById(R.id.btn3);
        btn4=findViewById(R.id.btn4);

        btn1.setOnClickListener(v -> {
            String valu1=edtvalu1.getText().toString();
            String valu2=edtvalu2.getText().toString();

            int v1=Integer.parseInt(valu1);
            int v2=Integer.parseInt(valu2);
            int result;
            result=v1+v2;
            txtResult.setText(result+"");
        });

        btn2.setOnClickListener(g -> {
            String valu1=edtvalu1.getText().toString();
            String valu2=edtvalu2.getText().toString();

            int v1=Integer.parseInt(valu1);
            int v2=Integer.parseInt(valu2);
            int result;
            result=v1-v2;
            txtResult.setText(result+"");
        });

        btn3.setOnClickListener(i -> {
            String valu1=edtvalu1.getText().toString();
            String valu2=edtvalu2.getText().toString();

            int v1=Integer.parseInt(valu1);
            int v2=Integer.parseInt(valu2);
            int result;
            result=v1*v2;
            txtResult.setText(result+"");
        });

        btn4.setOnClickListener(c -> {
            String valu1=edtvalu1.getText().toString();
            String valu2=edtvalu2.getText().toString();

            int v1=Integer.parseInt(valu1);
            int v2=Integer.parseInt(valu2);
            int result;
            result=v1/v2;
            txtResult.setText(result+"");
        });
    }
}

File: DateTimePicker/app/src/main/java/com/example/datetimepicker/MainActivity.kt
package com.example.datetimepicker

import android.app.DatePickerDialog
import android.os.Bundle
import android.widget.Button
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity
import java.util.*

class MainActivity : AppCompatActivity() {

    lateinit var pickDateBtn: Button
    lateinit var selectedDateTV: TextView

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        pickDateBtn = findViewById(R.id.idBtnPickDate)
        selectedDateTV = findViewById(R.id.idTVSelectedDate)

        pickDateBtn.setOnClickListener {
            val c = Calendar.getInstance()
            val year = c.get(Calendar.YEAR)
            val month = c.get(Calendar.MONTH)
            val day = c.get(Calendar.DAY_OF_MONTH)

            val datePickerDialog = DatePickerDialog(
                this,
                { view, year, monthOfYear, dayOfMonth ->
                    selectedDateTV.text =
                        (dayOfMonth.toString() + "-" + (monthOfYear + 1) + "-" + year)
                },
                year,
                month,
                day
            )
            datePickerDialog.show()
        }
    }
}
